### ISSUE TITLE
Support "composition entity"

### ISSUE THREAD

**Issue by daisy-ycguo** [openwhisk/openwhisk-wskdeploy#101]
Use case "[Data Processing with MessageHub](https://github.com/IBM/ogs-data-processing-messagehub)" needs an action sequence. We should support `composition entity` in wskdeploy.

Sample:
```
my_action_sequence:
    type: sequence
    sequence:
        - action_1
        - action_2
        - action_3
    inputs:
        simple_input_string: string
    outputs:
        annotated_output_string: string
```

**Comment by paulcastro** [comment]
Could you take a look at the openstack use case?  There is an example there for sequences also of the form:

```
 sequences:
    authorizedBackup: 
      actions: getApiToken, getServers, createBackup, callSlack
```

I wanted to have a separate entity for sequences since we may want to have more composition entities in the future.  What do you think?

I'm fine with having a formal YAML list or a comma delimited list to specify the actions that make up the sequence.

**Comment by daisy-ycguo** [comment]
I'm fine with your sample. I think yours looks simpler than the previous one which I copied from the pkg specification file.

@mrutkows do you agree to use a separate entity for sequence? If you agree, the specification file needs to be changed accordingly.

**Comment by paulcastro** [comment]
Sequence support is merged in PR #103

**Comment by daisy-ycguo** [comment]
Close this issue. Open another one to reflect the changes to spec.
